runners/direct-java/pom.xml

```diff
@@ -247,6 +247,14 @@
   </build>
 
   <dependencies>
+    <!-- jsr305 dependency must come before java-sdk-all, as java-sdk-all depends on
```

Comment:
this sounds like something that can have a better solution. I'd like to avoid cases like this.